classes-recipe/image_types_qcom.bbclass

```diff
@@ -30,13 +30,15 @@ create_qcomflash_pkg() {
     # esp image
     if [ -n "${QCOM_ESP_FILE}" ]; then
         install -m 0644 ${DEPLOY_DIR_IMAGE}/${QCOM_ESP_IMAGE}-${MACHINE}.rootfs.vfat ${QCOM_ESP_FILE}
+        ln -sf ${QCOM_ESP_IMAGE}-${MACHINE}.rootfs.vfat ${IMGDEPLOYDIR}/${QCOM_ESP_FILE}
```

Comment:
Wonder if it would be better to have the default link created by the same logic that creates the image, instead of having this link done when qcomflash gets generated.